File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/fs/modarchive/Author.java
/**
 * @file
 * @brief Author description POJO
 * @author vitamin.caig@gmail.com
 */

package app.zxtune.fs.modarchive;

public final class Author {

  public final int id;
  public final String alias;
  //TODO: pictures

  public Author(int id, String alias) {
    this.id = id;
    this.alias = alias;
  }

  //Currently API does not provide information about track without author info.
  //The only way to get them - perform search
  public static final Author UNKNOWN = new Author(0, "!Unknown");
}

File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/fs/modarchive/Author.kt
/**
 * @file
 * @brief Author description POJO
 * @author vitamin.caig@gmail.com
 */
package app.zxtune.fs.modarchive

//TODO: pictures
data class Author(val id: Int, val alias: String) {
    companion object {
        //Currently API does not provide information about track without author info.
        //The only way to get them - perform search
        @JvmField
        val UNKNOWN = Author(0, "!Unknown")
    }
}
